examples/http-server-oak-crud-middleware-with-sqlite3-db.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @title HTTP server: Performing CRUD operations using SQLite3
+ * @difficulty intermediate
+ * @tags web, cli, deploy
+ * @run -A <url>
+ * @group Network
+ *
+ * An example of a HTTP server for CRUD routes with oak middleware framework and SQLite3 database.It demonstrates the CRUD(Create, Read, Update and Delete) operations on file-based SQLite Database using HTTP methods (Get, Post, Put, Delete, Options)
+ */
+import { Application, Router } from "jsr:@oak/oak";
+import { Database } from "jsr:@db/sqlite";
+// Open a database from file, creates if doesn't exist. here the 'people.db' is a file-based database
+const peopleDb = new Database("people.db");
```

Comment:
have the database file be created in the user's temp directory, that way it'll automatically be cleaned up later

Reply:
or have it be created in the current directory

Reply:
or, and I'm a bigger fan of this option, create the database in memory using `:memory:` that way we don't have to worry about cleanup.

Reply:
thanks @RadhiRasho  for your feedback will update the code accordingly.